cdisc_rules_engine/data_service/sql_interface.py

```diff
@@ -121,6 +121,10 @@ def create_table(self, schema: SqlTableSchema) -> None:
         """Adds a table to the db"""
         create_stmt = SQLSerialiser.create_table_query_from_schema(schema)
         self.execute_sql(create_stmt)
+
+        id_column = SqlColumnSchema(name="id", hash="id", type="Num")
```

Comment:
We're probably better off adding this when any table schema is created, because we add the ID column to every table